buildout.cfg

```diff
@@ -255,49 +253,24 @@ on_update = true
 cmds =
     node_modules/.bin/lessc -ru --clean-css chsdi/static/less/extended.less chsdi/static/css/extended.min.css
 
-[ol3-install]
+[ol]
 recipe = collective.recipe.cmd
-on_install = true
-on_update = true
-cmds =
-    if ! [ -d chsdi/static/js/ol3 ];
-    then
-      cd ${buildout:directory}/chsdi/static/js/ && git clone https://github.com/geoadmin/ol3.git
-      cd ol3 && npm install && API_URL=${vars:api_url} ${buildout:bin-directory}/python build-ga.py
-      cd ${buildout:directory}
-    fi
-
-[ol3-serverconfig]
-recipe = collective.recipe.cmd
-on_install = true
-on_update = true
-cmds =
-    if  [ -d chsdi/static/js/ol3 ];
-    then
-      cd ${buildout:directory}/chsdi/static/js/ol3
-      rm -f ${buildout:directory}/chsdi/static/js/serverconfig*.js &&
-      API_URL=${vars:api_url} ${buildout:bin-directory}/python build-ga.py build/serverconfig &&
-      cp ${buildout:directory}/chsdi/static/js/ol3/build/*config*.js  ${buildout:directory}/chsdi/static/js/ &&
-      cd ${buildout:directory}
-    fi
-
-[ol3-update]
-recipe = collective.recipe.cmd
-on_install = true
+on_install = false
```

Comment:
I thinks we need to put this to `true`, otherwise buildout will fail the first time we do `buildout/bin/buildout install ol`.